packages/api/__tests__/index.js

```diff
@@ -209,7 +209,7 @@ describe('/verifyToken', () => {
   });
 });
 
-describe('/api/profile/createUser', () => {
+describe('/user', () => {
```

Comment:
Description should make it clear that these are tests for the *create/update* user route, not the *get* user route

<<<< describe('POST /user - creating and updating users', () => { >>>> would be better